README.md

```diff
@@ -1 +1,63 @@
-# PlyIO
+# Ply polygon file IO
+
+**PlyIO** is a package for reading and writing data in the
+[Ply](http://paulbourke.net/dataformats/ply/) polygon file format, also called
+the Stanford triangle format.
+
+## Quick start
+
+Here's an example of how to write a basic ply file containing random triangles
+and edges:
+
+```julia
+using PlyIO
+
+ply = Ply()
+push!(ply, PlyComment("An example ply file"))
+
+nverts = 1000
+
+# Random vertices with position and color
+vertex = PlyElement("vertex",
```

Comment:
Oh I get it. 

> that would still leave a naming decision for array vs list properties.

It would be really cool if the Julia commands to write a Ply file were one-to-one with the lines in the Ply header, which I think you've mostly achieved here. So to me whatever matches the ply text makes the most sense.